### ISSUE TITLE
[typescript-graphql-files-modules] Support documents that are not query or mutation in a trivial way

### ISSUE THREAD

**Issue by dobesv** [dotansimha/graphql-code-generator#1423]
**Is your feature request related to a problem? Please describe.**

I am using graphql-files-modules so that I can import graphql files from typescript without getting any complaints.

However, for the server-side schema files, this approach doesn't work.  It complains:

```
File "src/api/.../whatever.graphql" was filtered because it contains an invalid GraphQL document definition!
```

In my project I do use graphql imports on the server side.

**Describe the solution you'd like**

I could see two simple strategies here:

1. Output the each invalid module spec with a default export only, instead of skipping it
2. Maybe output some kind of catch-all at the end of the file for "*.graphql" with a default export only.

**Comment by dobesv** [comment]
*Workaround*

Can use the "add" plugin to insert a fallback declaration, and then exclude the supposedly "invalid" files from your file patterns.  For example:

```
 ./graphQLFilesModules.d.ts:
    documents:
      - 'src/{ui,lib,testing}/client/**/*.graphql'
    plugins:
      - add: |
          /* tslint:disable */
      - typescript-graphql-files-modules
      - add: |
          declare module '*.graphql' {
             const defaultDocument: string;
             export default defaultDocument;
          }
```

**Comment by dotansimha** [comment]
Yeah it should do the trick, but only for a single-document files, and you won't be able to import.
It might be a bit difficult to support this use-case, because the codegen plugins are getting the compiled schema, and not the pieces that created it. 
We might be able to add a flat that disables the document type validation, and this way all `.graphql` files will be loaded.

**Comment by dotansimha** [comment]
Fix available in 1.0.0 👍
